xenon/betterdiscordctl

```diff
@@ -0,0 +1,646 @@
+#!/usr/bin/env bash
+
+set -ueo pipefail
+shopt -s dotglob extglob nullglob
+
+# Constants
+VERSION=1.6.1
+SOURCE=$(readlink -f "${BASH_SOURCE[0]}")
+GITHUB_URL='https://raw.githubusercontent.com/bb010g/betterdiscordctl/master/betterdiscordctl'
```

Comment:
Oops, betterdiscordctl isn't right in place here. For some reason it was inside my local repository...